### PR TITLE
Se feedback 7

### PR DESCRIPTION
**Pull request by breijm**
Questions, Comments, Feedback - all are welcome
Last chance to make changes before the deadline Friday

### CODE REVIEW

File: se_feedback/sprint-7/feedback_sprint_7_all.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+| Code Review            |  10    |
+
+##Notes
+* ADD
+	* "Because the chrome storage is reliable" - can't argue with that logic, I would rephrase to the fact that your tool is as reliable as the Chrome Storage (cs). Let the reader draw there own conclusion on how reliable CS is.
+	* Same comments as last time about security (http, options without implementation, etc)
+* `GenericElementEventBinding`
+	* private attributes that are only accessed but never modified?
+* `MainController`
+	* line 99 + 142 seem to be creating a large amount of database objects?
```
Comment (MathiasMeuleman) [inline se_feedback/sprint-7/feedback_sprint_7_all.md]: @breijm would you have figured this out if it hadn't accidentally passed the DB-meeting? ;)

Comment (breijm) [inline se_feedback/sprint-7/feedback_sprint_7_all.md]: @MathiasMeuleman Probably not, but that's why I think meetings are important ;)

File: se_feedback/sprint-7/feedback_sprint_7_all.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+|------------------------|-------|
+| Branching              |  10    |
+| Code Review            |  10    |
+
+##Notes
+* ADD
+	* "Because the chrome storage is reliable" - can't argue with that logic, I would rephrase to the fact that your tool is as reliable as the Chrome Storage (cs). Let the reader draw there own conclusion on how reliable CS is.
+	* Same comments as last time about security (http, options without implementation, etc)
+* `GenericElementEventBinding`
+	* private attributes that are only accessed but never modified?
```
Comment (mpsijm) [inline se_feedback/sprint-7/feedback_sprint_7_all.md]: Do you mean `eventID` and `eventType`? Because they are set in the body of the constructor.

Comment (breijm) [inline se_feedback/sprint-7/feedback_sprint_7_all.md]: @mpsijm Yes I mean those two, they are never modified except for setting the value once in the constructor. In Java you can set attributes of this sort `public final`, it saves you a getter method per attribute.

**Comment by thervh70** [comment]
Reviewed and resolved. Will merge.
